RA_Integration/RA_User.h

```diff
@@ -117,3 +117,12 @@ class RAUsers
 	static LocalRAUser ms_LocalUser;
 	static std::map<std::string, RAUser*> UserDatabase;
 };
+
+
+// Shortcuts, these are too long to type out
+#define RA_USERNAME_LENGTH RAUsers::LocalUser().Username().length()
+#define RAUSER_NAME        RAUsers::LocalUser().Username()
+#define CRAUSER_NAME       NativeStr(RAUsers::LocalUser().Username()).c_str()
+#define RAUSER_TOKEN       RAUsers::LocalUser().Token()
+#define RAUSER_LOGON       RAUsers::LocalUser().IsLoggedIn()
```

Comment:
Again, you're sacrificing readability for space. Autocomplete should address your "too long to type out" concerns.

Reply:
Sure, but lots of the Windows API macros do exactly this, why should this be any different?

Reply:
http://www.stroustrup.com/bs_faq2.html#macro

Reply:
The "80 characters per line standard" is discussed here: https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width

I always thought it was so that you could print monospace code. Unless you are doing lots of printing out of code, I wouldn't worry about it ;)

Reply:
Sure, I don't like macros either. I'd rather use constexpr functions.
Thanks for the advice.